### ISSUE TITLE
sampling.sample_ppc should take a seed

### ISSUE THREAD

**Issue by alxempirical** [pymc-devs/pymc#1136]
Not having control over the stochasticity in `sample_ppc` can cause problems in a multiprocessing context.  E.g. in the code below the result `1` from the last line means that the exact same posterior predictive sample has been drawn in all four processes.

``` python
from concurrent import futures
from cPickle import load

from pymc3.sampling import sample_ppc

from fit import PKL_PATH
import model

TRACE = load(open(PKL_PATH))

def gensim(n):
    return sample_ppc(TRACE, samples=n)['walltime']

with futures.ProcessPoolExecutor() as executor:
    with model.model:
        probs = list(executor.map(gensim, [1] * 4))

print len(set(tuple(p[0]) for p in probs)) => 1
```

**Comment by twiecki** [comment]
Good point.

**Comment by aloctavodia** [comment]
Closing as this was solved in #1136
